intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/YangGrammar.bnf

```diff
@@ -258,13 +258,13 @@ type-stmt ::= TYPE_KEYWORD sep identifier-ref-arg-quoted optsep
 
 type-body-stmts ::= numerical-restrictions |
  decimal64-specification |
- string-restrictions |
  enum-specification |
  leafref-specification |
  identityref-specification |
  instance-identifier-specification |
  bits-specification |
- union-specification
+ union-specification |
+ string-restrictions
```

Comment:
try to keep .bnf as close to rfc as possible to keep it as easier to debug, so i would put string restrictions to its previous position

Reply:
Statements have to be in this order because string-restrictions can be empty so it have to be matched last.